Tools/CISupport/ews-build/steps.py

```diff
@@ -1765,8 +1765,8 @@ class ValidateCommitterAndReviewer(buildstep.BuildStep, GitHubMixin, AddToLogMix
     name = 'validate-commiter-and-reviewer'
     descriptionDone = ['Validated commiter and reviewer']
     VALIDATORS_FOR = {
-        # FIXME: This should be a bot, for now validation is manual
-        'apple': ['geoffreygaren', 'markcgee', 'rjepstein', 'JonWBedard', 'ryanhaddad', 'alancoon'],
+        # FIXME: Remove manual validators once bot is finished
```

Comment:
Nit: "finished" => "properly working"

Reply:
Is the name `webkit-bug-bridge` finalized?

Reply:
The name is not finalized, no. Although we've gotten pretty far with the hostname "bug-bridge.webkit.org", and our other GitHub accounts are prefixed with `webkit-`